CMakeSettings.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
+      "installRoot": "${projectDir}\\out\\install\\${name}",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "-v",
+      "ctestCommandArgs": "",
+      "variables": [
+        {
+          "name": "CMAKE_CXX_COMPILER",
+          "value": "D:/Logiciels/CMake/bin/cmake.exe",
+          "type": "FILEPATH"
+        },
+        {
+          "name": "CMAKE_C_COMPILER",
+          "value": "D:/Logiciels/CMake/bin/cmake.exe",
+          "type": "FILEPATH"
+        }
+      ]
+    }
+  ]
+}
```

Comment:
Will this be recognized by CMake?

Reply:
Deleted the file, turns out it was a residue from messing around with cmake 👍